### ISSUE TITLE
Having link to another card

### ISSUE THREAD

**Issue by Salokyn** [nextcloud/deck#1061]
**Describe the solution you'd like**
I'd like that writing `#NNN` (where NNN is the ID of a card) in a card description would create automatically a hypertex link that opens the card whose id is `NNN` when I click on it.
Hovering the link could show the title of the card `NNN`.

**Comment by juliushaertl** [comment]
Makes sense :+1:

**Comment by Salokyn** [comment]
To go a little further and navigate easily between linked cards, there could be a new tab in the the card details to show the cards which the actual one links to and those which have a link to the actual one.

**Comment by Octothorn** [comment]
To go further still, perhaps card links could have a category - one of "Blocker", "Blocked By", "Related" would be categories I would use today, but there could be others (perhaps user configurable).

**Comment by mulles** [comment]
If this feature would be realized, the foundation for a gantt chart https://github.com/nextcloud/deck/issues/928 representation would be laid.
After all it is about the possibility to link data together to make obvious that one card depends on the other. 

What would be the budget to fund such an enhancement?

**Comment by simonsan** [comment]
I think this would be a really worthwhile enhancement. Also, it would make it easy to have hierarchical cards, GANTT charts and many more. Each card could contain another field, which could be a `HashSet<LinkedCardIds>` that stores the linked cards. As card IDs are unique to each system, that could be enough for the start?

**Comment by xstable** [comment]
Currently there is a way to link to other Cards (even in other boards).
If you start to type `/` you got a pulldown menu where you can choose what you like to do. 

It work for linking cards, but in my case I found that the link isn't implemented in a useful way.
If I re-enter the card, I just see the http-Link-adress:

<img width="1132" height="1124" alt="Image" src="https://github.com/user-attachments/assets/073a2545-cf3f-4013-ab35-6291fa58c471" />

I think it would be much better to have some kind of clickable link. 
As it is now, you have to copy the URL and paste it in a new Tab (or do I miss something?!?)

**Comment by jospoortvliet** [comment]
So we introduced dependencies in this release, solving at least some of these ;-)
But being able to link by just typing # is a super cool idea!
